GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiableAccountDetail.m

```diff
@@ -16,5 +16,25 @@
 
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
 
+static NSString *const kAgeVerificationScope = @"https://www.googleapis.com/auth/verified.age.over18.standard";
```

Comment:
Probably should update this to `kAccountDetailTypeAgeOver18Scope` since there may be other ages that are verifiable in the future.

Reply:
Done.